recipe/patch_yaml/keyring.yaml

```diff
@@ -0,0 +1,16 @@
+# from this bit of code
+# if (record_name == "keyring" and
+#         record["version"] == "23.6.0" and
+#         record["build_number"] == 0):
+#     for i, dep in enumerate(record["depends"]):
+#         dep_name = dep.split()[0]
+#         if dep_name == "importlib_metadata" and ">=" not in dep:
+#             record["depends"][i] = "importlib_metadata >=3.6"
+if:
+  name: keyring
+  version: "23.6.0"
+  build_number: 0
+then:
+  - replace_depends:
+      old: "importlib_metadata?( *)[!>=]*"
+      new: "importlib_metadata >=3.6"
```

Comment:
```suggestion
      new: "importlib_metadata >=3.6"

```

Reply:
I'll be committing during the next few hours to this PR, and then I'll run the pre-commit hooks, consider it WIP til I mark it as ready for review. Sorry for the noise and thanks! 🙏